myapp/app/models/task.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :bigint           not null, primary key
+#  title       :string(255)
+#  description :string(255)
+#  priority    :integer
+#  status      :integer
```

Comment:
for priority and status, you probably will have few fixed values and for that maybe `tinyint` is enough.

we can do something like `t.integer :priority, limit: 1`


For reference

```

:limit	    Numeric Type	Column Size	Max value
1	     tinyint	         1 byte	          127
2	     smallint	         2 bytes	32767
3	     mediumint	         3 byte	        8388607
nil, 4, 11    int(11)	         4 byte	       2147483647
5..8	      bigint	         8 byte	     9223372036854775807
```

Reply:
didn't know about this either. Thanks for sharing~